binaries/Readme.md

```diff
@@ -111,24 +111,90 @@ Binaries (query engine binary and migration engine binary) are at the core of Ph
-| linux-glibc-libssl1.0.2-ubuntu1604 |         Ubuntu 16.04          | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-glibc-libssl1.0.2-ubuntu1604/prisma.gz) | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/migration-engine.gz)  | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/prisma-fmt.gz)  |
-| linux-glibc-libssl1.1.0-ubuntu1604 |         Ubuntu 16.04          | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-glibc-libssl1.1.0-ubuntu1604/prisma.gz) | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/migration-engine.gz)  | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/prisma-fmt.gz)  |
-| linux-glibc-libssl1.1.1-ubuntu1604 |         Ubuntu 16.04          | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-glibc-libssl1.1.1-ubuntu1604/prisma.gz) | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/migration-engine.gz)  | [download](https://s3-eu-west-1.amazonaws.com/prisma-native/master/latest/linux-musl-libssl1.1.1/prisma-fmt.gz)  |
+Prisma provides multiple binaries for various platforms and operating systems.
+
+### Why?
+
+We need different binaries because operating systems such as Windows and Linux work 
+fundamentally different. Some Linux distributions have the same issue. On top of that,
+different versions of Linux distributions have different versions of OpenSSL installed,
+which is why we need different binaries for those as well.
+
+### How?
+
+We build multiple binaries on common operating systems with different combinations
+of OpenSSL versions. This results in a few binaries which work on a large selection
+of operating systems, distributions and cloud platforms.
+
+Initially, we built specific binaries for given platforms (e.g. Netlify), which resulted in many 
+binaries which couldn't be shared with similar platforms and it was hard to maintain. 
+
+### Binary builds
+
+The following table lists the pre-built binaries provided by Prisma.
+These include Windows and Mac, and multiple variations for Linux distributions
+
+| Build          | OS                  | OpenSSL variations | Downloads |
+| :------------- | :------------------ | :----------------: | :-------- |
+| `darwin`       | Mac                 | n/a                | (tbd)     |
+| `windows`      | Windows             | n/a                | (tbd)     |
+| `debian`       | Debian              | 1.0.x, 1.1.y       | (tbd)     |
+| `rhel`         | CentOS              | 1.0.x, 1.1.y       | (tbd)     |
+
+### Linux distributions and versions
+
+This table shows which Linux distributions are compatible with our pre-built binaries.
```

Comment:
Confirm if the minor version variations work across compiled binaries? 

Looks like a yes from an initial discussion with Dom ✅